.github/ISSUE_TEMPLATE/TASK.yaml

```
@@ -1,11 +1,75 @@
----
-name: 🏃🏻‍♂️ Task
-description: A general task that does not fit into other categories.
-title: "[task]: "
-type: Task
+name: "Task"
+description: "Work item that goes through refinement before development"
+title: "[Task] <short title>"
+labels: ["task"]
+
 body:
   - type: textarea
-    id: description
+    id: purpose
     attributes:
-      label: Description
-      description: Give a brief description of the task.
+      label: Purpose
+      description: Briefly explain why this task is important and what it delivers.
+      placeholder: |
+        Briefly describe what the task should deliver and why it matters.
+        e.g. Add caching to CI pipelines to reduce build time.
+    validations:
+      required: true
+
+  # --- Do we want to estimate the size of the task? If yes, this is my suggestion,
+  # since we used T-shirt sizes in the old DevOps team and it worked well.
+  # This combines that approach with a bit of planning poker. ---
+  - type: dropdown
+    id: size
+    attributes:
+      label: Size (T-shirt)
+      options: [XS, S, M, L, XL]
+    validations:
+      required: true
+
+  - type: textarea
+    id: acceptance
+    attributes:
+      label: Acceptance Criteria
+      description: What must be true for the task to be considered complete.
+      placeholder: |
+        Examples:
+        - [ ] When the pipeline runs, the new caching is used and reduces build time.
+        - [ ] When the feature flag is enabled, the new UI is visible to all users.
+        - [ ] When Stefan can press the button without getting an error, we are done.
```

Others will be able to see these, so we probably want to reword it a little ;-)

```suggestion
        - [ ] When a user can press the green button without getting an error.
```